src/vue/modules/coinpayments-deposit/components/pending-issuances-table.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <table class="deposit-table">
+    <pending-issuances-table-row
+      v-for="(item, index) in pendingIssuances"
+      :end-time="item.endTime"
+      :amount="item.amount"
+      :asset="item.asset"
+      :address="item.address"
+      :key="index"
+    />
+  </table>
+</template>
+
+<script>
+import PendingIssuancesTableRow from './pending-issuances-table-row'
+
+export default {
+  name: 'deposit-table',
+  components: {
+    PendingIssuancesTableRow,
+  },
+  props: {
+    pendingIssuances: { type: Array, required: true },
+  },
+}
+</script>
+
+<style lang="scss">
+  @import '~@scss/variables';
+  .deposit-table {
```

Comment:
The rows here are look like garbage because there’s no newlines

Reply:
ok